src/main/groovy/com/palantir/gradle/javadist/tasks/CreateStartScriptsTask.groovy

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2015 Palantir Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * <http://www.apache.org/licenses/LICENSE-2.0>
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.gradle.javadist.tasks
+
+import com.palantir.gradle.javadist.DistributionExtension
+import com.palantir.gradle.javadist.JavaDistributionPlugin
+import org.gradle.api.GradleException
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.jvm.application.tasks.CreateStartScripts
+
+class CreateStartScriptsTask extends CreateStartScripts {
```

Comment:
if we want to kill it, I'd do it in a separate PR.
this would primarily break windows users as go-java-launcher doesn't currently work on windows.